websites/B/Broken Picturephone/presence.ts

```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+    clientId: "756196794727399617"
+});
+
+presence.on("UpdateData", async () => {
+
+    const browsingStamp = Math.floor(Date.now() / 1000);
+
+    const button = document.getElementsByTagName('button');
+    const valor = button.length;
+    const players = document.getElementsByClassName('userActive');
+    const nump = `(${players.length} of 6 players)`;
+
+        const data: PresenceData = {
```

Comment:
merge consts

Reply:
how i do it? sorry i'm starting with ts

Reply:
const var1 = obj,
var2 = obj,
var3 = obj;